personality/ui_gsissh/config.pan

```diff
@@ -0,0 +1,123 @@
+unique template personality/ui_gsissh/config;
+
+#base configuration
+
+"/software/components/chkconfig/service/gsisshd/on" = "";
+"/software/components/chkconfig/service/gsisshd/startstop" = true;
+"/software/components/gsissh/server/port" = GSISSH_PORT;
+
+# Change the GLOBUS SXXsshd file, that is read by ncm-gsissh, in order to 
+#   - use gsisshd name
+#   - SOURCE the environment before starting the server
+
+variable CONTENTS = <<EOF;
+#!/bin/sh
+#
+# Init file for OpenSSH server daemon for GRIF interrsite logins
+#
+# chkconfig: 2345 99 25
+# description: OpenSSH server daemon for GRIF intersite logins
+#
+
+#this is a sh script : don't assume environment is defined, especially when using "service" to restart services
+#and globus things really need the globus environment
+ENV_FILE="/etc/profile.d/env.sh"
+if [ -f $ENV_FILE ]; then
+         source $ENV_FILE
+else
+        echo "error : could not source environment for GLOBUS initialisation ($ENV_FILE)" >&2
+        exit 2
+fi
+
+export GRIDMAPDIR=/etc/grid-security/gridmapdir
+
+GLOBUS_LOCATION="/opt/globus"
+export GLOBUS_LOCATION
+
+. ${GLOBUS_LOCATION}/libexec/globus-script-initializer
+. ${libexecdir}/globus-sh-tools.sh
+
+PID_FILE=${localstatedir}/sshd.pid
+EOF
+
+variable CONTENTS = CONTENTS + 'SSHD_ARGS="-p ' + to_string(GSISSH_PORT) + '"' + "\n" + <<EOF;
```

Comment:
It would be much nicer to either write a sysconfig file and source it or use `substitute()` to inject this value into the script.